src/test/rgw/rgw_multi/zone_rados.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+try:
+    from itertools import izip_longest as zip_longest
+except ImportError:
+    from itertools import zip_longest
+
+from nose.tools import eq_ as eq
+
+from rgw_multi.multisite import *
+
+log = logging.getLogger(__name__)
+
+def check_object_eq(k1, k2, check_extra = True):
```

Comment:
we could also actually override operator `__eq__` for class k1, so just eq(k1,k2) will work, but maybe for later